src/SFML/Graphics/ImageLoader.cpp

```diff
@@ -40,8 +40,7 @@ namespace
     // Convert a string to lower case
     std::string toLower(std::string str)
     {
-        for (std::string::iterator i = str.begin(); i != str.end(); ++i)
-            *i = static_cast<char>(std::tolower(*i));
+        std::transform(str.begin(), str.end(), str.begin(), ::tolower);
```

Comment:
https://stackoverflow.com/a/37438120

I believe we should use lambda as adviced here

Reply:
Why? we've no compilation issue

Reply:
There are standard libs which have overloads for tolower